modules/apps/27-interchain-accounts/keeper/keeper.go

```diff
@@ -163,3 +171,22 @@ func (k Keeper) SetInterchainAccountAddress(ctx sdk.Context, portID string, addr
 	store := ctx.KVStore(k.storeKey)
 	store.Set(types.KeyOwnerAccount(portID), []byte(address))
 }
+
+// NegotiateAppVersion handles application version negotation for the IBC interchain accounts module
+func (k Keeper) NegotiateAppVersion(
```

Comment:
Moved the implementation of this from `AppModule` to the Keeper in order to gain access to the `accountKeeper` without having to expose it.
AppModule now just invokes this func on the Keeper.